### DISCUSSION TITLE
Only sync latest data via docker?

### DISCUSSION THREAD

**Discussion by marty-sullivan** [open-meteo/open-meteo#723] [Q&A] [answered]
Hi, I'm following the instructions in the various README files to sync the ncep_hrrr_conus_15min dataset.

I'm confused as to why the documents seem to say to use the `--past-days 1` argument. Is there a way to simply specify to only sync the latest data from the s3 bucket and nothing else? I'm mostly confused as to why I would ever want to have data synced that is any older than latest, and maybe the single previous forecast data. However, maybe I'm not understanding a valid reason as to why this is documented in the README. Can someone provide some more context?

Here is the command I crafted now that works fine, but seems to sync a lot more data than I actually need:

`docker run -it --rm -v contentsum_open-meteo-data:/app/data ghcr.io/open-meteo/open-meteo sync ncep_hrrr_conus_15min categorical_freezing_rain,diffuse_radiation,frozen_precipitation_percent,precipitation,shortwave_radiation,temperature_2m,visibility,wind_gusts_10m,wind_u_component_10m,wind_u_component_80m,wind_v_component_10m,wind_v_component_80m --past-days 1`

**Comment by patrick-zippenfenig** [comment, marked as answer]
Hi,
Open-Meteo is using a time series database to optimise access for individual locations. Data is organised in chunks of 100-300 time steps and 10-20 locations in order for compression to work efficiently. Therefore each single file contains 2-5 days of weather data.

| File | Time |
| - | - |
| chunk_1234.om | Jan 1. - Jan 5. |
|chunk_1235.om | Jan 6. - Jan 10. | 

With every model update, an entire file needs to updated and in some cases, even two "time chunks" and therefore two files. This is inefficient to write and transfer, but it is incredibly fast to read. High compression ratios reduce this inefficiency. A single update of the open-meteo database is still significantly smaller than a single set of GRIB files from the original weather services.

Due to this design, it is impossible to only download the latest data. Data from the previous days always needs to be present.

If you only need s small subset of data consider using the public API service instead of downloading the entire database files.

**Reply by marty-sullivan** [reply]
ok, thank you for clarifying
